source/slang/parser.cpp

```diff
@@ -95,9 +95,9 @@ namespace Slang
         RefPtr<StructDecl>					ParseStruct();
         RefPtr<ClassDecl>					    ParseClass();
         RefPtr<Stmt>					ParseStatement();
-        RefPtr<Stmt>			        ParseBlockStatement();
-        RefPtr<DeclStmt>			ParseVarDeclrStatement(Modifiers modifiers);
-        RefPtr<IfStmt>				ParseIfStatement();
+        RefPtr<Stmt>			        parseBlockStatement();
+        RefPtr<DeclStmt>			parseVarDeclrStatement(Modifiers modifiers);
+        RefPtr<IfStmt>				parseIfStatement();
```

Comment:
I appreciate the cleanup, but it helps a lot if you make each PR only do one thing. A separate “fixed for naming convention” PR is really easy to review, and streamlined the actual new code PR.

Reply:
Yeah I realized that after I made some of these changes and I don't feel like reverting them either.